airflow/providers/cncf/kubernetes/operators/kubernetes_pod.py

```diff
@@ -138,6 +138,9 @@ class KubernetesPodOperator(BaseOperator):  # pylint: disable=too-many-instance-
     :type init_containers: list[kubernetes.client.models.V1Container]
     :param log_events_on_failure: Log the pod's events if a failure occurs
     :type log_events_on_failure: bool
+    :param retry_only_on_pod_launching_failure: retry logic is only effective if pod launching fails. This
+        prevents retries when image failures occurs. Useful for non-idempotent tasks.
```

Comment:
Looks like it should be changed like this.
```suggestion
        prevents retries when image failures occurs. Useful for idempotent tasks.
```